www/caddy/+POST_DEINSTALL.post

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
the idea of the file is ok, and we know about this issue, but essentially you end up removing all files when the package is reinstalled too. especially the syslog-ng restarts are not wanted and the files left mean  no harm to the system either. I'd recommend to remove.

Reply:
Deleted the file, you're right about that.